components/org.wso2.carbon.aws/pom.xml

```diff
@@ -0,0 +1,93 @@
+        <artifactId>identity-userstore-aws</artifactId>
+        <relativePath>../../pom.xml</relativePath>
+        <version>1.0.0-SNAPSHOT</version>
+    </parent>
+
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>org.wso2.carbon.user.aws</artifactId>
+    <packaging>bundle</packaging>
+    <name>WSO2 Carbon - AWS Cloud Directory service over the REST based User Manager</name>
+    <url>http://wso2.org</url>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.wso2.carbon</groupId>
+            <artifactId>org.wso2.carbon.user.core</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.wso2.apache.httpcomponents</groupId>
+            <artifactId>httpclient</artifactId>
+            <version>${httpclient.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>com.googlecode.json-simple</groupId>
+            <artifactId>json-simple</artifactId>
+            <version>${json.simple.version}</version>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-scr-plugin</artifactId>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <extensions>true</extensions>
+                <configuration>
+                    <instructions>
+                        <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
+                        <Private-Package>org.wso2.carbon.aws.user.store.mgt.internal</Private-Package>
+                        <Import-Package>
+                            javax.sql,
+                            org.apache.axiom.om.util; version="${axiom.osgi.version.range}",
+                            org.apache.commons.logging; version="${commons-logging.osgi.version.range}",
+                            org.osgi.framework; version="${osgi.framework.imp.pkg.version.range}",
+                            org.osgi.service.component; version="${osgi.service.component.imp.pkg.version.range}",
+                            org.wso2.carbon.user.api; version="${carbon.user.api.imp.pkg.version.range}",
+                            org.wso2.carbon.user.core.*;version="${carbon.kernel.imp.pkg.version.range}",
+                            org.wso2.carbon.context.*;version="${carbon.kernel.imp.pkg.version.range}",
+                            org.apache.http.*;version="${httpclient.version}",
+                            org.apache.commons.*,
+                        </Import-Package>
+                        <Export-Package>
+                            !org.wso2.carbon.aws.user.store.mgt.internal,
+                            org.wso2.carbon.aws.*; version="${identity.userstore.aws.exp.version}",
+                            org.json.simple.*; version="${json.simple.version}",
```

Comment:
Why do we need to export org.json.simple ?

Reply:
If I havn't export this , I'm getting error as
java.lang.ClassNotFoundException: org.json.simple.parser.ParseException cannot be found by org.wso2.carbon.userstore.aws_1.0.0.SNAPSHOT

Reply:
We don't encourage embedded dependencies. Check whether this package is there in our OSGI environment or not. If not you may need to pack the respective orbit bundle